### ISSUE TITLE
publish to npm

### ISSUE THREAD

**Issue by michielbdejong** [ripple/ilp-plugin-xrp-escrow#6]
https://www.npmjs.com/package/ilp-plugin-xrp-escrow has not been published yet.

**Comment by sharafian** [comment]
@justmoon has the permissions for CI and the NPM deploy key

**Comment by emschwartz** [comment]
I think this is done now

**Comment by michielbdejong** [comment]
It should be done again (ideally, automate this on circleci?), for v1.0.7 (npm still shows v1.0.5).

**Comment by sharafian** [comment]
yeah, it was automated on CI but looks like it stopped. @justmoon can you take a look, maybe it was removed from circle?
